samples/sobel_old/sobel_main.py

```diff
@@ -0,0 +1,56 @@
+from PIL import Image
+import heterocl as hcl
+import numpy as np
+import math
+import imageio
+from urllib.request import urlopen
```

Comment:
This one is supposed to be the version optimized with FIFO channels. You can keep it here for now. I will update this example later.